app/models/miq_report/search.rb

```diff
@@ -92,7 +92,19 @@ def paged_view_search(options = {})
 
     search_options = options.merge(:class => db, :conditions => conditions, :include_for_find => includes)
     search_options.merge!(:limit => limit, :offset => offset, :order => order) if order
-    search_options[:extra_cols] = va_sql_cols if va_sql_cols.present?
+
+    extra_cols = if includes
```

Comment:
Kinda seems like this should be put in it's own method.  `va_sql_cols` is just a memoized method as well, so you could toss it there, but I could see the argument that this should be it's own method and probably makes sense that it is.

Reply:
I am also confused why this is needed, which leads back to LJ's comment here:

https://github.com/ManageIQ/manageiq/pull/18176#issuecomment-437186993

The `:extra_cols` attribute on `Rbac` (\*\*cough\*\* \*\*cough\*\* nick is the original author \*\*cough\*\* \*\*cough\*\*) was meant to add additional columns and not remove ones that were already there (even if the default `Arel.star` was originally just assumed).  So something seems borked in Rails if we need to start adding the foreign keys ourselves, or we broke something in `manageiq` along the way.

Is there possibly an additional `.select` in this report that is causing the issue?  I also have some changes in https://github.com/ManageIQ/manageiq/pull/18353 that are affecting the results for `:extra_cols` that might be beneficial or detrimental to what you have here as well.